visidata/guides/MenuGuide.md

```diff
@@ -0,0 +1,24 @@
+# The Menu System
+
+Located at the very top of the screen, the menu organizes many commands within submenu trees.
+
+Navigate the menu like a GUI by using the mouse to click on menu items.
```

Comment:
Our phrasing is, "Click on menu items to navigate the menu like a GUI".  Always "Input X to do Y".